src/schema/v2/fair.ts

```diff
@@ -227,7 +227,12 @@ export const FairType = new GraphQLObjectType<any, ResolverContext>({
             args,
             { marketingCollectionsLoader }
           ) => {
-            args.slugs = kaws_collection_slugs
+            args.slugs = kaws_collection_slugs || []
```

Comment:
I wouldn't mutate the args.slugs, I'd rather use a new variable to avoid any side effects

Reply:
good one

Reply:
good one

Reply:
good one

Reply:
good one

Reply:
good one

Reply:
good one

Reply:
wow my connection is really bad

Reply:
wow my connection is really bad